nilearn/surface.py

```diff
@@ -0,0 +1,572 @@
+"""
+Functions for surface manipulation.
+"""
+import warnings
+
+from distutils.version import LooseVersion
+
+import numpy as np
+import scipy
+from scipy import sparse, interpolate
+import sklearn.preprocessing
+
+# These will be removed (see comment on _points_in_unit_ball)
```

Comment:
I could not find _points_in_unit_ball ?

I am trying to understand why it is there now and why it will be removed after PR is ready to merge ?